### PR TITLE
Update students.yml

### PR DESCRIPTION
**Pull request by DotSlashTX**
Updated the studentss list 
Name: Prasenjit Gautam

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -78,7 +78,9 @@
   lat: 28.641485
   lng: 77.371386
 
-- name: Ngô Hồng Phúc
-  image: oem.jpg
-  github: Jerry2001
```
Comment (abishekvashok) [inline _data/students.yml:83]: Please don't remove an existing student.

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Also add your image.

**Comment by DotSlashTX** [comment]
I'm facing problem in uploading the image

**Comment by akshatnitd** [comment]
@FreakinBabaJi I can see that you have not added the photo before pushing. Make sure to ```git add <photo-path>``` before pushing. Also, do not remove any existing student.

**Comment by ba11b0y** [comment]
@FreakinBabaJi Please do not remove the existing student, it is causing a merge conflict. Please add yourself without removing anyone else.

**Comment by DotSlashTX** [comment]
I'm not removing any existing student . I'm simply adding my name but I
don't know why this conflict is showing up

**Comment by ba11b0y** [comment]
Please check your students.yml in your fork here https://github.com/FreakinBabaJi/gci17.fossasia.org/blob/patch-1/_data/students.yml
It shows that you have removed an existing student.
Don't worry, just edit again and make a commit.
Good going

**Review by ba11b0y — CHANGES_REQUESTED** [review]
Please make the changes

**Comment by abishekvashok** [comment]
Conflicts, pls resolve them

**Comment by DotSlashTX** [comment]
I'm unable to get what conflicts are actually there? Please help me

**Comment by abishekvashok** [comment]
@FreakinBabaJi sure, conflicts arise when yourchanges giff was overwritten by a recent commit. So you ome into a conflict. You have a conflict in _data/students.yml. To resolve conflicts pls follow this guide:
http://tedfelix.com/software/git-conflict-resolution.html
http://genomewiki.ucsc.edu/index.php/Resolving_merge_conflicts_in_Git

**Pull request closed without merging**